### PR TITLE
Fixes for Example object #906

### PR DESCRIPTION
**Pull request by darrelmiller**

### CODE REVIEW

File: versions/3.0.md [hunk 1]
```diff
@@ -765,7 +765,7 @@ Field Name | Type | Description
...
 <a name="parameterSchema"></a>schema | [Schema Object](#schemaObject) | The schema defining the type used for the parameter.
-<a name="parameterExamples"></a>examples | [Examples Array](#examplesArray) | Examples of the content type.  Each example in the Examples Array must be in the correct format as specified parameter encoding.  The `examples` object is mutually exclusive to the `example` object.  Furthermore, if referencing a `schema` which contains an example, the `examples` value will _override_ the example provided by the schema.
+<a name="parameterExamples"></a>examples | [[Example Object](#exampleObject)] | Examples of the content type.  Each example in the Examples Array must be in the correct format as specified parameter encoding.  The `examples` object is mutually exclusive to the `example` object.  Furthermore, if referencing a `schema` which contains an example, the `examples` value will _override_ the example provided by the schema.
```
Comment (fehguy) [inline versions/3.0.md]: _must_ ==> _should_

File: versions/3.0.md [hunk 2]
```diff
@@ -765,7 +765,7 @@ Field Name | Type | Description
...
 <a name="parameterExample"></a>example | [Example Object](#exampleObject) | Example of the content type.  The example object _must_ be in the correct format as specified in the parameter encoding.  The `example` object is mutually exclusive to the `examples` object.  Furthermore, if referencing a `schema` which contains an example, the `example` value will _override_ the example provided by the the schema.
```
Comment (fehguy) [inline versions/3.0.md]: _must_ ==O _should_

File: versions/3.0.md (hunk 1 above)
Comment (webron) [inline versions/3.0.md]: SHOULD

File: versions/3.0.md (hunk 2 above)
Comment (webron) [inline versions/3.0.md]: SHOULD

File: versions/3.0.md [hunk 3]
```diff
@@ -1092,8 +1092,8 @@ Each content type object provides schema and examples for a the media type ident
...
-<a name="contentTypeExample"></a>example | [Example Object](#exampleObject) | Example of the content type.  The example object _should_ be in the correct format as specified in the _content_ type.  The `example` object is mutually exclusive to the `examples` object.  Furthermore, if referencing a `schema` which contains an example, the `example` value will _override_ the the example provided by the schema.
+<a name="contentTypeExamples"></a>examples | [Examples Array](#examplesArray) | Examples of the content type.  Each example in the Examples Array SHOULD be in the correct format as specified in the _content_ type.  The `examples` object is mutually exclusive to the `example` object.  Furthermore, if referencing a `schema` which contains an example, the `examples` value will _override_ the example provided by the schema.
```
Comment (webron) [inline versions/3.0.md]: Should be modified to [[Example Object](#exampleObject)] like above.

File: versions/3.0.md [hunk 4]
```diff
@@ -765,8 +765,8 @@ Field Name | Type | Description
...
-<a name="parameterExample"></a>example | [Example Object](#exampleObject) | Example of the content type.  The example object _must_ be in the correct format as specified in the parameter encoding.  The `example` object is mutually exclusive to the `examples` object.  Furthermore, if referencing a `schema` which contains an example, the `example` value will _override_ the example provided by the the schema.
+<a name="parameterExamples"></a>examples | [[Example Object](#exampleObject)] | Examples of the content type.  Each example in the Examples Array SHOULD be in the correct format as specified parameter encoding.  The `examples` object is mutually exclusive to the `example` object.  Furthermore, if referencing a `schema` which contains an example, the `examples` value will _override_ the example provided by the schema.
```
Comment (webron) [inline versions/3.0.md]: - Examples array (not Examples Array)
- value SHALL _override_ the example (and not value will _override_ the example). This one is less critical.

Changes should be applied to other example(s) descriptions.

File: versions/3.0.md [hunk 5]
```diff
@@ -1646,38 +1646,32 @@ X-Rate-Limit-Reset:
     type: integer
 ```
 
-#### <a name="examplesObject"></a>Examples Object
+#### <a name="exampleObject"></a>Example Object
 
 Allows sharing examples for operation requests and responses.
```
Comment (webron) [inline versions/3.0.md]: Since the fields table is dropped, consider explaining this is a free form object (possibly allowing primitive values as well?).

**Pull request merged**